fix(genapi): update quota documentation (#5056)
scaleway/docs-content: pages/generative-apis/troubleshooting/fixing-common-issues.mdx

### Solution
- Smooth out your API requests rate by limiting the number of API requests you perform over a given minute so that you remain below your [Organization quotas for Generative APIs](/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
- [Add a payment method](/billing/how-to/add-payment-method/#how-to-add-a-credit-card) and [validate your identity](/account/how-to/verify-identity/) to increase automatically your quotas [based on standard limits](/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
- Reduce the size of the input or output tokens processed by your API requests.
- Use [Managed Inference](/managed-inference/), where these quotas do not apply (your throughput will be only limited by the amount of Inference Deployment your provision)

> - Contact your existing Scaleway account manager or [our Sales team ](https://www.scaleway.com/en/contact-sales/) to discuss about volume commitment for specific models that will allow us to increase your quota proportionnaly.

Reviewer: ```suggestion
- Contact your assigned Scaleway account manager or [our Sales team](https://www.scaleway.com/en/contact-sales/) to discuss volume commitments for specific models, which will enable us to increase your quota proportionally.
```
